### PR TITLE
Adding endpoint override for SDK methods

### PR DESCRIPTION
**Pull request by lukeocodes**

### CODE REVIEW

File: src/transcription/preRecordedTranscription.ts [hunk 1]
```diff
@@ -77,7 +78,7 @@ export const preRecordedTranscription = async (
     apiKey,
     apiUrl,
     requireSSL,
-    `/v1/listen?${querystring.stringify(transcriptionOptions)}`,
+    `/${endpoint}?${querystring.stringify(transcriptionOptions)}`,
```
[the remark this answers is not in the document]
Comment (lukeocodes) [reply src/transcription/preRecordedTranscription.ts:81]: If we make version part of the SDK code, and trating it as part of a customisable endpoint, then the API becomes tightly bound to that URL structure.

Also, given our request to support sandbox, that would also stop us from doing that.

**Comment by jpvajda** [comment]
@lukeocodes if we are offering this override how will we document it so users understand it's use?

**Comment by lukeocodes** [comment]
> @lukeocodes if we are offering this override how will we document it so users understand it's use?

Something along the lines of "for instances you may want to override an endpoint". We don't need to document the options, not until we have previous versions to support. 

Things we want to do: 
- Support internal requirements for using endpoints not publically documented

Things we don't want to do:
- Define fixed naming convensions such as versions in URLs making arbitrary endpointing impossible
- Hardcode the options, sign-posting endpoints that might not have DDoS mitigation

File: package.json [hunk 2]
```diff
@@ -29,9 +29,17 @@
     "sdk"
   ],
   "author": {
-    "name": "Michael Jolley",
-    "email": "michael.jolley@deepgram.com"
+    "name": "DevRel Team",
```
Comment (jpvajda) [inline package.json:32]:
```suggestion
    "name": "Deepgram DevRel Team",
```

**Review by jpvajda — COMMENTED** [review]
Just had one suggestion

**Review by briancbarrow — APPROVED** [review]

**Pull request merged**